apollo/plugin.py

```diff
@@ -146,14 +147,26 @@ def _parse_people_enrichment_data(enrichment_data: dict) -> ApolloPersonProfile:
         departments=enrichment_data.get("departments", []),
         subdepartments=enrichment_data.get("subdepartments", []),
         seniority=enrichment_data.get("seniority"),
+        phone_numbers=[
+            number.get("sanitized_number", "")
+            for number in enrichment_data.get("phone_numbers", [])
+        ],
     )
 
 
 def apollo_people_enrichment(
-    first_name: str, last_name: str, email: str
+    first_name: Optional[str] = None,
+    last_name: Optional[str] = None,
+    email: Optional[str] = None,
+    name: Optional[str] = None,
+    organization_name: Optional[str] = None,
+    domain: Optional[str] = None,
 ) -> ApolloPersonProfile:
     """Retrieves a person's profile through Apollo's People Enrichment API,
-    including details such as employment, social media urls, and other personal data."""
+    including details such as employment, social media urls, and other personal data.
+    You can either use the combination of first and last name or a single name
```

Comment:
i think if you add a newline here, the second part will only appear in the doc string, not the description.